pkg/gardenlet/controller/seed/seed_extension_control.go

```diff
@@ -17,22 +17,20 @@ package seed
 import (
 	"fmt"
 
-	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
-	gardencorev1beta1helper "github.com/gardener/gardener/pkg/apis/core/v1beta1/helper"
+	api "github.com/gardener/gardener/pkg/apis/core/v1beta1"
```

Comment:
Why was this renamed? Can we keep it `gardencorev1beta1`?

Reply:
Next time when we change the version to `v1` everything needs to be replaced.

Reply:
Yes, but I still find it WAY more clear and better understandable reading the code when the API version is used as import alias (`corev1.Secret, appsv1.StatefulSet`, ...). Sometimes there are other structs with the same name like a resource, and then it's not always clear where this comes from.